src/Data/Unrestricted/Linear.hs

```diff
@@ -70,6 +70,8 @@ module Data.Unrestricted.Linear
     Consumable (..),
     Dupable (..),
     Movable (..),
+    RepStream (..),
```

Comment:
I don't think that we should export `RepStream`'s constructor. It's better left as an abstract type.

Also, since we export it, we probably want to give it a better name still (I do recognise that `RepStream` is significantly better than `Streamish` :smile: ).